### DISCUSSION TITLE
not enough token when sending sol

### DISCUSSION THREAD

**Discussion by theking200794** [phantom/docs#170] [Help & Support] [answered]
### Summary

I have 0.11159 sol in my account and I am trying to send 10000 lamports using the below method
```
 transactionInstructions.push(
          createTransferInstruction(
            fromAccount.address, // source
            associatedTokenTo, // dest
            publicKey,
            10000
          )
        );
```
     I am using spl token lib in react for sending spl tokens. iot, and usdc sending through phantom working fine but on sending sol it gives this error.
     

<img width="335" alt="Screenshot 2024-02-02 at 5 36 55 PM" src="https://github.com/phantom/docs/assets/5977965/1cfc349b-2fc6-48fc-aae1-27115726cf3e">

### Steps to Reproduce

after getting associatedTokens for sender and receiver using 
```
 const associatedTokenFrom = await getAssociatedTokenAddress(
          mintToken,
          publicKey
        );
```
when I am hitting the below method 
```
 transactionInstructions.push(
          createTransferInstruction(
            fromAccount.address, // source
            associatedTokenTo, // dest
            publicKey,
            10000
          )
        );
```
phantom is throwing an error that I don't have enough tokens. even though I have  0.11159 sol in my wallet.
but when I hit the send button in my app iot, usdc sending works fine but on sending sol Phantom is throwing the error `not enough tokens`. 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hi @theking200794,

This likely has to do with the lamport calculation parameter in your createTransferInstruction. I don't believe this is related to Phantom.

I would investigate that parameter. If the issue doesn't persist with other wallets, please let me know.
